### PR TITLE
fix: correct loc of comments, add offset into loc point

### PR DESCRIPTION
**Pull request by JounQin**
related https://github.com/mdx-js/eslint-mdx/issues/380

### Initial checklist

*   [x] I read the support docs 
*   [x] I read the contributing guide 
*   [x] I agree to follow the code of conduct 
*   [x] I searched issues and couldn’t find anything (or linked relevant results below) 
*   [x] If applicable, I’ve added docs and tests

### Description of changes

As title

### CODE REVIEW

File: packages/micromark-util-events-to-acorn/dev/index.js [hunk 1]
```diff
@@ -146,7 +146,28 @@ export function eventsToAcorn(events, options) {
...
+      assert('end' in comment, 'expected `end` in comment from acorn')
+
+      const pointStart = parseOffsetToUnistPoint(comment.start)
+      const pointEnd = parseOffsetToUnistPoint(comment.end)
+      comment.start = pointStart.offset
+      comment.end = pointEnd.offset
+      comment.loc = {
+        start: {
+          line: pointStart.line,
+          column: pointStart.column - 1,
```
Comment (JounQin) [inline packages/micromark-util-events-to-acorn/dev/index.js:160]: image

Seems incorrect, let me check 😅

Comment (JounQin) [reply packages/micromark-util-events-to-acorn/dev/index.js:160]: It seems this lib is not called at all for these comments. @wooorm Where do you think could be related else? 🤣

**Comment by JounQin** [comment]
It seems it has been handlered incorrectly in `visit`.

**Comment by wooorm** [comment]
I don’t understand what you did? Why did you stop? Is this project not used in `eslint-mdx`? Which `visit` do you mean?

**Pull request closed without merging**